app/src/main/java/com/projectit3048c/MainActivity.kt

```diff
@@ -58,10 +58,9 @@ class MainActivity : ComponentActivity() {
     @Composable
     fun TextFieldWithDropdownUsage(dataIn: List<Food>?, label : String = "", take :Int = 3) {
 
-        val dropDownOptions = remember { mutableStateOf(listOf<Food>()) }
-        val textFieldValue = remember {mutableStateOf(TextFieldValue()) }
-        val dropDownExpanded = remember { mutableStateOf(false) }
-
+        val dropDownOptions = remember { mutableStateOf(listOf<Food>())}
+        val textFieldValue = remember {mutableStateOf(TextFieldValue())}
+        val dropDownExpanded = remember { mutableStateOf(false)}
```

Comment:
Removed unnecessary spaces between colons and parenthesis suggested from: https://kotlinlang.org/docs/coding-conventions.html#colon

Reply:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.